blog/models.py

```diff
@@ -1,50 +1,46 @@
 from django.db import models
+from django.db.models import Count
 from django.conf import settings
 from django.db.models.constraints import UniqueConstraint
 from accounts.models import Account
 import datetime
 from django.template.defaultfilters import slugify
+# from unidecode import unidecode
 from django.urls import reverse
 
+
 class Category(models.Model):
     name = models.CharField(max_length = 50)
     slug = models.SlugField(unique=True,blank=True,null=True)
     
     def save(self, *args, **kwargs):
-         if not self.id:
+        if not self.id:
                 # Newly created object, so set slug
             self.slug = slugify(self.name)
-            super(Category, self).save(*args, **kwargs)
+        super(Category, self).save(*args, **kwargs)
+    # def __str__(self):
+    #     str(self.name)
 class Post(models.Model):
     title = models.CharField(max_length=100)
-    slug  = models.SlugField(null=True,blank=True, unique=True)
+    slug  = models.SlugField(max_length = 100,null=True,blank=True, unique=True)
```

Comment:
Your coding style is not consistent. The line above is `max_length=100` and this line is `max_length = 100`. I would prefer `max_length=100`.

Reply:
Let's refer https://www.python.org/dev/peps/pep-0008/ for coding style. You can use other tools like Pycharm to auto-format code.

Reply:
Thanks for replying